src/nav/ChatNavBar.js

```diff
@@ -36,8 +36,10 @@ class ChatNavBar extends PureComponent<Props> {
             dispatch(navigateBack());
           }}
         />
-        <Title color={color} narrow={narrow} />
-        <TitleNavButtons color={color} narrow={narrow} />
+        <View style={styles.navContent}>
```

Comment:
This `view` wrapper makes easy to understand the components structure/arrangements.